osu.Game.Rulesets.Catch/UI/CatcherArea.cs

```diff
@@ -221,9 +221,9 @@ private void beginTrail()
             /// Add a caught fruit to the catcher's stack.
             /// </summary>
             /// <param name="fruit">The fruit that was caught.</param>
-            public void Add(DrawableHitObject fruit)
+            public void PlaceOnPlate(DrawableHitObject fruit)
             {
-                float ourRadius = fruit.DrawSize.X / 2 * fruit.Scale.X;
+                float ourRadius = ((DrawableCatchHitObject)fruit).DisplayRadius;
```

Comment:
Looks like the arg type can be scoped down to `DrawableCatchHitObject` and the cast removed without any other alterations.

Reply:
nice catch